README.md

```diff
@@ -1,159 +1,184 @@
+
+### Contributing to MLJ
+
+MLJ is an ambitious project and we need all the help we can get!
+There are multiple ways you can contribute.
+The table below helps indicate where you can help and what are the subjective requirements in terms of Julia and ML expertise.
+
+Julia | ML         | What to do
+----- | ---------- | ----------
+=     | =          | use MLJ and give us feedback, help us write better tutorials, suggest missing features, test the less mature model packages
++     | =          | package to facilitate visualising results in MLJ
++     | +          | add/improve data pre-processing tools
++     | +          | add/improve interfaces to other model-providing packages
++     | +          | functionalities for time series
++     | +          | functionalities for systematic benchmarking of models
+++    | =          | decrease the overhead incurred by MLJ
+++    | +          | add parallelism and/or multithreading to MLJ
++     | ++         | add  interface with probabilistic programming packages (*there is an ongoing effort to interface with [Soss.jl](https://github.com/cscherrer/Soss.jl)*)
+++    | ++         | more sophisticated HP tuning (BO, Bandit, early stopping, ...) possibly as part of a external package(s), possibly integrating with Julia's optimisation and autodiff packages
+
+If you're interested in one of these beyond the first one, please get in touch with either Anthony Blaom or Thibaut Lienart on Slack and we can further guide you.
+Thank you!
+
+---
+
+### Models available
+
+There is a wide range of models accessible via MLJ.
+We are always looking for contributors to add new models or help us test existing ones.
+The table below indicates the models that are accessible at present along with a subjective indication of how mature the underlying package is.
+
+* *experimental*: indicates the package is fairly new and/or is under active development; you can help by testing these packages and making them more robust,
+* *medium*: indicates the package is fairly mature but may benefit from optimisations and/or extra features; you can help by suggesting either,
+* *high*: indicates the package is very mature and functionalities are expected to have been fairly optimised and tested.
+
+| Package | Models | Maturity | Note
+| ------- | ------ | -------- | ----
+[Clustering.jl] | KMeans, KMedoids | high | [note]
+[DecisionTree.jl] | DecisionTreeClassifier, DecisionTreeRegressor | high | [note]
+[GaussianProcesses.jl] | GPClassifier | medium | [note]
+[GLM.jl] | LinearRegressor, LinearBinaryClassifier, LinearCountRegressor | medium | [note]
+[LIBSVM.jl] | LinearSVC, SVC, NuSVC, NuSVR, EpsilonSVR, OneClassSVM | high | also via ScikitLearn.jl
+[MLJModels.jl] (builtins) | StaticTransformer, FeatureSelector, FillImputer, UnivariateStandardizer, Standardizer, UnivariateBoxCoxTransformer, OneHotEncoder, ConstantRegressor, ConstantClassifier, (KNNRegressor) | medium |
```

Comment:
Why is KNNRegressor here? Is this a reference to the obsolete version? I think we can just drop this.